src/components/Card/Card.tsx

```diff
@@ -28,6 +29,7 @@ type CardProps = {
   isCurrent?: boolean;
   isLocked?: boolean;
   currentLabel?: string;
+  redirectUrl?: string;
```

Comment:
Let's just call this `url`. The `redirect` prefix might imply some sort of redirect vs navigation and be a bit confusing